sample/kotlin-timber/src/main/kotlin/com/datadog/android/sample/webview/WebFragment.kt

```diff
@@ -29,8 +29,10 @@ class WebFragment : Fragment() {
     ): View? {
         val rootView = inflater.inflate(R.layout.fragment_web, container, false)
         webView = rootView.findViewById(R.id.webview)
-        webView.webViewClient = RumWebViewClient()
-        webView.webChromeClient = RumWebChromeClient()
+        webView.webViewClient =
```

Comment:
And why changing that ?

Reply:
Same thing...editor's problem...will put this back.